Editor/AGS.Editor/Utils/Validation.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using System.Text.RegularExpressions;
+
+namespace AGS.Editor.Utils
+{
+    public class Validation
+    {
+        public static bool FilenameIsValid(string filename)
+        {
+            return !(filename.IndexOfAny(Path.GetInvalidFileNameChars()) >= 0);
+        }
+
+        public static bool PathDoesNotExist(string path)
+        {
+            if (File.Exists(path))
+                return false;
+
+            if (Directory.Exists(path))
+                return false;
+
+            return true;
+        }
+
+        public static bool PathIsAbsoluteDriveLetter(string path)
+        {
+            return Regex.IsMatch(path, @"^[a-zA-Z]:\\");
+        }
+
+        public static bool PathIsAbsolute(string path)
+        {
+            bool rooted = false;
+
+            try
+            {
+                if (Path.IsPathRooted(path))
+                    rooted = true;
+            }
+            catch (ArgumentException)
+            {
+                // pass
```

Comment:
I wanted to mark that the lack of handling is intended, so that it is clear that the intention was to do nothing, not that someone forgot to add some code here (similar to labelling fall-through on a traditional case statement).